tests/test_gndtruth_smoketest-json-bz2.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "4",
+  "include": [
+    "config_gndtruth_smoketest.json"
+  ],
+  "controllers": [
+    {
+      "binding": "gndtruth_extractor",
+      "vehicle": "default",
+      "args": {
+        "output_type": "json.bz2",
+        "components": [
+          "cloe::gndtruth_lane_sensor",
+          "cloe::default_world_sensor"
+        ]
+      }
+    }
+  ]
+}
```

Comment:
Git is warning here that there is no final newline. Best thing is to configure your editor to always add them.